src/components/tree-item/tree-item.tsx

```diff
@@ -0,0 +1,291 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Element,
+  Prop,
+  State,
+  Listen,
+  h,
+  getAssetPath,
+  Watch
+} from "@stencil/core";
+@Component({
+  tag: "gxg-tree-item",
+  styleUrl: "tree-item.scss",
+  shadow: true,
+  assetsDirs: ["tree-item-assets"]
+})
+
+// return "./card-assets/new-card.svg";
+export class GxgTreeItem {
+  //PROPS
+  /**
+   * Set this attribute if you want the gxg-treeitem to display a checkbox
+   */
+  @Prop() checkbox = false;
+
+  /**
+   * Set this attribute if you want the gxg-treeitem checkbox to be checked by default
+   */
+  @Prop() checked = false;
+
+  /**
+   * Set this attribute when you are downloading a resource
+   */
+  @Prop() downloading = false;
+
+  /**
+   * Set thhe left side icon from the available Gemini icon set : https://gx-gemini.netlify.app/?path=/story/icons-icons--controls
+   */
+  @Prop() leftIcon: string;
+  /**
+   
+
+  /**
+   * Set thhe right side icon from the available Gemini icon set : https://gx-gemini.netlify.app/?path=/story/icons-icons--controls
+   */
+  @Prop() rightIcon: string;
+  /**
+
+  /**
+   * If this tree-item has a nested tree, set this attribute to make the tree open by default
+   */
+  @Prop() open = false;
+
+  //STATE
+  @Prop() isLeaf: boolean;
+  @State() paddingLeft = "4px";
+  @State() numberOfParentTrees = 1;
+  @State() height = 0;
+  @State() parentHasCheckbox = false;
```

Comment:
Doesn't seem to be used. Are you planning to use it in the future? What would it be used for?

Reply:
You are right.
The reason is on the begining, they seemed to be neccesary, but finally they didn't, and I forgot to remove them.
The implementation of this component was not a straight line, and it wasn't easy to me to retain in my mind all the variables that were once used/usefull. Thank you for keeping an eye on this. 

 numberOfParentTrees is neccesary however, to calculate the padding left of the items.